src/utils/hooks/useGetReviewInfo.tsx

```diff
@@ -100,7 +100,7 @@ const useGetReviewInfo = ({ applicationId, serial, skip = false }: UseGetReviewI
 
       const assignment: AssignmentDetails = {
         id,
-        reviewer: reviewer as User,
+        reviewer: reviewer as User & { id: number },
```

Comment:
For some reason, the "id" field is defined as optional in Gql definitions. So this is effectively an `as` for a single field.